platforms/gke/base/use-cases/federated-learning/examples/nvflare-tff/README.md

```diff
@@ -0,0 +1,281 @@
+you need about 8GB of persistent storage space and can take up to 20 minutes.
+For a production deployment, consider using Cloud Build.
+
+To run the script, do the following:
+
+1. Open [Cloud Shell](https://cloud.google.com/shell).
+
+1. Run the script to build the container image:
+
+   ```sh
+   "${ACP_PLATFORM_BASE_DIR}/use-cases/federated-learning/examples/nvflare-tff/build-container-image.sh"
+   ```
+
+### Create NVIDIA FLARE deployment descriptors
+
+In this section, you create descriptors to deploy NVIDIA FLARE in the reference
+architecture:
+
+1. Open [Cloud Shell](https://cloud.google.com/shell).
+
+1. Create a directory to store the NVIDIA FLARE workspace:
+
+   ```bash
+   mkdir "${HOME}/nvflare-workspace"
+   ```
+
+1. Grant the NVIDIA FLARE user ownership on the NVIDIA FLARE workspace:
+
+   ```bash
+   sudo chown -R 10000:10000 "${HOME}/nvflare-workspace"
+   ```
+
+1. Load NVIDIA FLARE example environment configuration:
+
+   ```bash
+   source "${ACP_PLATFORM_BASE_DIR}/use-cases/federated-learning/examples/nvflare-tff/setup-environment.sh" && load_fl_terraform_outputs
+   ```
+
+1. Run an NVIDIA FLARE container based on the container image you built in the
+   preceding section:
+
+   ```bash
+   docker run --rm -v "${HOME}/nvflare-workspace:/opt/NVFlare/workspace" -it "${NVFLARE_EXAMPLE_CONTAINER_IMAGE_LOCALIZED_ID_WITH_TAG}" bash
+   ```
+
+1. Run the NVIDIA FLARE provisioning tool to create a NVIDIA FLARE project
+   configuration file (`project.yml`) that you can customize to your needs:
+
+   ```bash
+   nvflare provision
+   ```
+
+   When prompted, choose the non highly-available deployment option because
+   highly-available NVIDIA FLARE deployments are not yet supported on
+   Kubernetes.
+
+1. Run the provisioning tool again to generate deployment descriptors:
+
+   ```bash
+   nvflare provision
+   ```
+
+   The provisioning tool generates deployment descriptors:
+
+   - `server1` is the server that will aggregate all the results from the
+     computation
+   - `site-1` and `site-2` are the clients that will be connected to the server
+   - `admin@nvidia.com` is the administration client to start and list jobs
+
+1. Exit the NVIDIA FLARE container:
+
+   ```bash
+   exit
+   ```
+
+1. Copy the workspace folder to Cloud Storage:
+
+   ```bash
+   gcloud storage cp --recursive "${HOME}/nvflare-workspace/workspace" "gs://${NVFLARE_EXAMPLE_WORKSPACE_BUCKET_NAME}"
+   ```
+
+### Push the example container image
+
+In this section, you run a script that pushes the container images to the
+Artifact Registry repository that the Federated learning reference architecture
+provides.
+
+To run the script, do the following:
+
+1. Open [Cloud Shell](https://cloud.google.com/shell).
+
+1. Run the script to build the container image:
+
+   ```sh
+   "${ACP_PLATFORM_BASE_DIR}/use-cases/federated-learning/examples/nvflare-tff/push-container-image.sh"
+   ```
+
+### Verify that NVIDIA FLARE pods are running
```

Comment:
This seems somewhat redundant with the `Get the list of NVIDIA FLARE pods running in the cluster:` below. You could probably remove this section and just move the GKE Workloads Dashboard as another option below.

Reply:
removed